test/Posix.hs

```diff
@@ -90,12 +90,8 @@ operationParent =
         (parent ($(mkAbsDir "/foo") </>
                     $(mkRelDir "bar")) `shouldReturn`
          $(mkAbsDir "/foo"))
-     it "parent \"\" == \"\""
-        (parent $(mkAbsDir "/") `shouldReturn`
-         $(mkAbsDir "/"))
-     it "parent (parent \"\") == \"\""
-        ((parent $(mkAbsDir "/") >>= parent) `shouldReturn`
-         $(mkAbsDir "/"))
+     it "cannot take the parent of the root directory" $
```

Comment:
We should instead use `shouldThrow` here and check that it throws the correct thing, not discard the info checking against `Nothing`. I'll push a fix.